apps/rewards/app/components/App/App.js

```diff
@@ -193,20 +228,21 @@ class App extends React.Component {
       },
       panelTitle,
     })
+    this.props.selectReward(this.state.selected, String(reward.rewardId))
   }
 
   claimReward = reward => {
     // TODO
     this.props.api.claimReward(reward.rewardId + reward.claims).toPromise()
   }
-
+  /*
```

Comment:
This is not needed, for the PR to be considered complete we should clean these kind of comments and dead code